cirq-superstaq/cirq_superstaq/ops/qubit_gates.py

```diff
@@ -829,9 +872,17 @@ def _json_dict_(self) -> Dict[str, Any]:
         return cirq.obj_to_dict_helper(self, ["rz_rads"])
 
 
-def custom_resolver(  # pylint: disable=missing-function-docstring
-    cirq_type: str,
-) -> Union[Type[cirq.Gate], None]:
+def custom_resolver(cirq_type: str) -> Union[Type[cirq.Gate], None]:
+    """Tells `cirq.to_json` and `cirq.read_json` how to handle custom cirq-superstaq's custom
```

Comment:
nit: resolvers are only used in `read_json` i think

Reply:
Right. Fixed.